packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/mfaoob/nextsteps.md

```diff
@@ -0,0 +1,3 @@
+Now that you have implemented authorization in your app, you can add features such as:
+
+Brand customization ([custom domain](/docs/guides/custom-url-domain/), [custom SMS messages](/docs/guides/custom-sms-messaging/), [custom emails](/docs/guides/custom-url-domain/main/#about-custom-email-notification-domains), and [custom error pages](/docs/guides/custom-error-pages/)).
```

Comment:
I'm not sure any of these actions make sense with direct auth. Do we need this Next Steps section?